src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -54,14 +56,16 @@ public OverseasEntitySubmissionDtoValidator(EntityNameValidator entityNameValida
         this.trustCorporateValidator = trustCorporateValidator;
         this.historicalBeneficialOwnerValidator = historicalBeneficialOwnerValidator;
         this.updateValidator = updateValidator;
+        this.removeValidator = removeValidator;
     }
 
     public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
         if (isRoeUpdateEnabled && overseasEntitySubmissionDto.isForUpdate()) {
             validateFullUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
-        } else if (overseasEntitySubmissionDto.isForRemove()) {
+        }  else if (overseasEntitySubmissionDto.isForRemove()) {
```

Comment:
Please remove the extra space introduced before the `else` statement.

Reply:
space removed

Reply:
space removed